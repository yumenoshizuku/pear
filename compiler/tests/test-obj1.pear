Foo(bar) (
    fb1 = 22,
    fb2 = 69,
    return bar,
)
Main() (
    tryme = Foo(),
    puts(tryme(12)),
    puts(tryme(8)),
    puts(tryme.fb1),
    puts(tryme.fb2),
)

