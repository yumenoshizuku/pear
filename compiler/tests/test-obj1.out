#include <stdio.h>
#include <stdlib.h>
#include <gtk/gtk.h>
#include <glib.h>
#include <glib/gstdio.h>


int main(int argc, char** argv)
{
	
	gtk_init(&argc, &argv);
	printf("%d\n", 12);
	printf("%d\n", 8);
	printf("%d\n", 22);
	printf("%d\n", 69);
	gtk_main();
	return 0;
}

